tbx/core/blocks.py

```diff
@@ -100,6 +102,28 @@ class Meta:
 class PullQuoteBlock(StructBlock):
     quote = CharBlock(form_classname="quote title")
     attribution = CharBlock()
+    company_logo = ImageChooserBlock(
```

Comment:
**Nitpick (non-blocking)**: Wondering whether this should be just `logo`, since this could, for instance, be a brand or product logo.

Reply:
Ah, I just did this to match the front-end but can change it